packages/@aws-cdk/aws-cloudformation-codepipeline/README.md

```diff
@@ -0,0 +1,50 @@
+## AWS CodePipline Actions for AWS CloudFormation
+
+This module contains an Action that allows you to use CloudFormation actions from CodePipeline.
```

Comment:
README requires a bit more work 😄 Where are the gifs?

Also, I think it will be useful to list all the action types and when would someone want to use them